Breeze.UI/src/app/wallet/history/history.component.ts

```diff
@@ -80,39 +100,34 @@ export class HistoryComponent implements OnInit, OnDestroy {
             }
           }
         }
-      )
-    ;
+      );
   };
 
-  private getTransactionInfo(transactions: any) {
-    this.transactions = [];
+  private getTransactionInfo(transactions: Array<ITransaction>) {
+    this.transactions = transactions.map(transaction => new TransactionInfo(transaction));
+  }
 
-    for (const transaction of transactions) {
-      let transactionType;
-      if (transaction.type === 'send') {
-        transactionType = 'sent';
-      } else if (transaction.type === 'received') {
-        transactionType = 'received';
-      }
-      const transactionId = transaction.id;
-      const transactionAmount = transaction.amount;
-      let transactionFee;
-      if (transaction.fee) {
-        transactionFee = transaction.fee;
+  public onBlur() {
+    this.isSearching = this.searchForm.get('transactionId').value ? true : false;
```

Comment:
Change to `!!this.searchForm.get('transactionId').value`, remove ` ? true : false`